src/bench/duplicate_inputs.cpp

```diff
@@ -26,7 +26,9 @@ static void DuplicateInputs(benchmark::Bench& bench)
     CMutableTransaction coinbaseTx{};
     CMutableTransaction naughtyTx{};
 
-    CBlockIndex* pindexPrev = ::ChainActive().Tip();
+    LOCK(cs_main);
```

Comment:
21767: this LOCK() is from some another PR

Reply:
Resolved in latest push